src/libs/actions/Welcome.ts

```diff
@@ -26,21 +37,21 @@ let currentUserAccountID;
  * - Whether we have loaded all reports the server knows about
  */
 function checkOnReady() {
-    if (!_.isBoolean(isFirstTimeNewExpensifyUser) || isLoadingReportData) {
+    if (isFirstTimeNewExpensifyUser === undefined || isLoadingReportData) {
```

Comment:
I might be missing, but I think `!_.isBoolean(isFirstTimeNewExpensifyUser)` and `isFirstTimeNewExpensifyUser === undefined` are not the same thing

Reply:
ah okay that makes sense 👍 

the previous code was quite confusing 😅